apps/passport-server/src/routing/routes/frogcryptoRoutes.ts

```diff
@@ -115,6 +115,11 @@ export function initFrogcryptoRoutes(
     );
   });
 
+  app.get("/frogcrypto/scores", async (req, res) => {
+    const result = await frogcryptoService.getScores();
+    res.json(result);
+  });
```

Comment:
maybe worth naming this and propagating the naming convention of `scoreBoard` over `scores`